src/Test/Core/PluginUpdateCheckerTest.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Nikse.SubtitleEdit.Plugins;
+
+namespace Test.Core
+{
+    [TestClass]
+    public class PluginUpdateCheckerTest
+    {
+        [TestMethod]
+        public async Task CheckAsyncTest()
+        {
+            var pluginPath = Environment.GetEnvironmentVariable("plugin_directory", EnvironmentVariableTarget.User);
```

Comment:
Looking for a solution to this...
- Built one of the plugin fiel with decreased version!

Should we let the test cases call internet?